datamodel_code_generator/imports.py

```diff
@@ -26,11 +26,12 @@ class Imports(DefaultDict[Optional[str], Set[str]]):
     def __str__(self) -> str:
         return self.dump()
 
-    def __init__(self) -> None:
+    def __init__(self, use_exact: bool = False) -> None:
         super().__init__(set)
         self.alias: DefaultDict[Optional[str], Dict[str, str]] = defaultdict(dict)
         self.counter: Dict[Tuple[Optional[str], str], int] = defaultdict(int)
         self.reference_paths: Dict[str, Import] = {}
+        self.use_exact: bool = use_exact
```

Comment:
Do we need the `self.use_exact` ?

Reply:
It is needed because the method below `__change_from_import` has no access to instance attributes since it is a classmethod. So, the attribute is needed here to propagate the flag all the way down to that classmethod where it is actually needed.